clients/js/cmds/generate.ts

```diff
@@ -290,7 +290,9 @@ function parseAddress(chain: ChainName, address: string): string {
   } else if (chain === "osmosis") {
     throw Error("OSMOSIS is not supported yet");
   } else if (chain === "sui") {
-    throw Error("SUI is not supported yet");
+    if (/^(0x)?[0-9a-fA-F]+$/.test(address)) {
```

Comment:
What is the material difference between this entire method and this one used in the sdk? If they are the same, we should standardize usage to one of them; I don't have a strong opinion on which https://github.com/wormhole-foundation/wormhole/blob/6def9b306ccbf001185436e985ca58b551ea6e60/sdk/js/src/utils/array.ts#L214

Reply:
it does look like there is duplicate functionality